packages/tables/src/Actions/Action.php

```diff
@@ -22,6 +22,11 @@ public function getLivewireCallMountedActionName(): string
         return 'callMountedTableAction';
     }
 
+    public function getAlpineClickHandler(): ?string
+    {
+        return "mountTableAction('{$this->getName()}')";
```

Comment:
I think we really need to avoid sending the selected records to the backend if we don't need them, so do we need to add in a condition here to make sure that is the case?

Reply:
Yeah, something like that

Reply:
Done!